src/index.ts

```diff
@@ -58,6 +61,7 @@ export default class DeclarativePDF {
     this.defaults = new PaperDefaults(opts?.defaults);
     this.normalize = opts?.normalize;
     this.debug = opts?.debug ?? {};
+    this.documentTitle = opts?.documentTitle ?? 'document';
```

Comment:
nesto ovog tipa:

```suggestion
    if (opts.document.meta) this.documentMeta = opts.document.meta;
```

i onda na liniji 126 (zapravo mi je sad tek sjelo zasto si me pitao za taj edge case) trebamo gledat postoji li `this.documentMeta` - i ako postoji, moramo to loadati u PDFDocument i setat sve sto je setable.